.gitpod.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM gitpod/workspace-mysql
+
+USER root
+
+# Update APT Database
+### base ###
+RUN sudo apt-get update -q \
+    && sudo apt-get install -y php-dev
+
+# Install XDebug
+RUN wget http://xdebug.org/files/xdebug-2.9.1.tgz \
+    && tar -xvzf xdebug-2.9.1.tgz \
+    && cd xdebug-2.9.1 \
+    && phpize \
+    && ./configure \
+    && make \
+    && sudo mkdir -p /usr/lib/php/20190902 \
+    && sudo cp modules/xdebug.so /usr/lib/php/20190902 \
+    && sudo bash -c "echo -e '\nzend_extension = /usr/lib/php/20190902/xdebug.so\n[XDebug]\nxdebug.remote_enable = 1\nxdebug.remote_autostart = 1\n' >> /etc/php/7.4/cli/php.ini"
```

Comment:
No need to open a subshell here: this line could be replaced with:

```sh
printf 'zend_extension=xdebug\n[XDebug]\nxdebug.remote_enable=1\nxdebug.remote_autostart=1\n' > "$(php-config --ini-dir)/xdebug.ini"
```

Reply:
I have taken over so far. But then I followed the existing linking of the other extensions. thanks for you feedback